validators/node-api.md

```diff
@@ -0,0 +1,178 @@
+# Node REST API v1
+
+Each SKALE node has `skale_api` container with Flask webserver that provides REST JSON API to interact with it.  
+SKALE node REST API is available locally on the node and could be accessed on http://localhost:3007.
+
+> ❗️ WARNING: REST API provides an access to private methods such as money transfers, node controls, etc. Make sure that port `3007` is available only to your trusted clients.
+
+> Node REST API v1 available in `skale_api > 2.0.0-develop.0`
+
+## Table of contents
+
+- [Node REST API v1](#node-rest-api-v1)
+  * [Table of contents](#table-of-contents)
+  * [API structure](#api-structure)
+  * [Public/private APIs](#public-private-apis)
+  * [REST JSON API](#rest-json-api)
+    + [Logs](#logs)
+      - [Methods](#methods)
+      - [Usage](#usage)
+    + [Node](#node)
+      - [Methods](#methods-1)
+    + [Usage](#usage-1)
+    + [Health](#health)
+      - [Methods](#methods-2)
+      - [Usage](#usage-2)
+    + [sChains](#schains)
+      - [Methods](#methods-3)
+      - [Usage](#usage-3)
+    + [SSL](#ssl)
+      - [Methods](#methods-4)
+      - [Usage](#usage-4)
+    + [Wallet](#wallet)
+      - [Methods](#methods-5)
+      - [Usage](#usage-5)
+
+## API structure
```

Comment:
I think _Endpoint structure_ is a little more accurate.

Reply:
👍